File: API/Models/Product.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API.models
{
    public class Product
    {
        public int ProductId {get; set; }
        public string ProductName {get; set; }
        public string Description {get; set; }
        public double Price {get; set; } // 100.00 => 10000

        public string PictureUrl {get; set; }

        public string Type {get; set; }
        public int QuantityInStock {get; set; }
    }
}

File: react-client/src/app/models/product.ts
export interface Product {
  productId: number;
  productName: string;
  description: string;
  price: number;
  pictureUrl: string;
  type?: string;
  quantityInStock?: number;
}

export interface ProductParams {
  orderBy: string;
  keyWord?: string;
  types: string[];
  pageNumber: number;
  pageSize: number;
}
